.github/workflows/upload-and-deploy.yaml

```diff
@@ -48,21 +48,60 @@ jobs:
         uses: aws-actions/configure-aws-credentials@v1
         with:
           aws-region: us-east-1
-          role-to-assume: arn:aws:iam::914833433684:role/sagebase-github-oidc-sage-bion-ProviderRolerecover-VTOKECAKJUBV
+          role-to-assume: ${{ vars.AWS_CREDENTIALS_IAM_ROLE }}
           role-session-name: GitHubActions-${{ github.repository_owner }}-${{ github.event.repository.name }}-${{ github.run_id }}
           role-duration-seconds: 1200
 
       - name: Setup sam
         uses: aws-actions/setup-sam@v2
 
       - name: Set namespace for non-default branch or for tag
-        if: github.ref_name != 'main'
         run: echo "NAMESPACE=$GITHUB_REF_NAME" >> $GITHUB_ENV
 
-      - name: Copy files to templates bucket, use dev cloudformation bucket if not main branch
-        if: github.ref_name != 'main'
-        run: python src/scripts/manage_artifacts/artifacts.py --upload --namespace $NAMESPACE --cfn_bucket recover-dev-cloudformation
+      - name: Copy files to templates bucket, use dev cloudformation bucket
+        run: python src/scripts/manage_artifacts/artifacts.py --upload --namespace $NAMESPACE --cfn_bucket ${{ vars.CFN_BUCKET }}
+
+
+  upload-files-main:
```

Comment:
Per our huddle, let's table the production deployment until we have the deployment in the test account completely figured out.  Please comment this out so we don't lose all the effort you put in.

Reply:
I think I'm just going to update the environment of this `upload-files-main` to be `develop`. This is then saying that if the PR is good to go and gets merged into main, then the deployment of the main branch will upload artifacts to the develop account, guaranteeing us the latest version of the artifacts in the dev account bucket but with no namespace attached/it won't get deleted when cleanup happens as well in the future. Hope this is an okay resolution.

Reply:
Removed `upload-files-main` job block and instead added the updating of the non-namespaced `recover-dev-cloudformation/recover/src` and namespaced artifacts bucket to under the `upload-files` job block